**Windows** [Build 4336](http://brontes.lab.eng.brq.redhat.com/viewLog.html?buildId=89545&buildTypeId=WF_PullRequest_Windows) outcome was **FAILURE** using a merge of b7b94bb4c7f9605bf9fb86cd0bd277c4e71748b8
Summary: Tests failed: 1 (1 new), passed: 3329, ignored: 193 Build time: 00:57:28
### Failed tests
```
org.jboss.as.test.clustering.cluster.web.passivation.CoarseSessionPassivationTestCase(SYNC-tcp).test: java.lang.AssertionError: response doesn't contain header 'result', all response headers = {name=Connection, value=keep-alive}, {name=X-Powered-By, value=Undertow/1}, {name=Set-Cookie, value=JSESSIONID=P-b9VgqHx9HgXxZx0w4ntcVpn2z_9H1G6MW_cNC3.node-0; path=/passivation}, {name=Server, value=WildFly/10}, {name=D7eSClpBmWeC3g5j0u643yV9NlTcpHYb1SQAK8EE, value=PASSIVATION}, {name=Content-Length, value=0}, {name=jsessionid, value=P-b9VgqHx9HgXxZx0w4ntcVpn2z_9H1G6MW_cNC3}, {name=Date, value=Fri, 05 Feb 2016 22:30:20 GMT}, 
	at org.jboss.as.test.clustering.cluster.web.passivation.SessionPassivationTestCase.getRequiredHeaderValue(SessionPassivationTestCase.java:207)
	at org.jboss.as.test.clustering.cluster.web.passivation.SessionPassivationTestCase.test(SessionPassivationTestCase.java:153)
------- Stdout: -------
23:30:03,256 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Starting deployment=NONE, container=container-0
23:30:04,259 WARNING [org.jboss.as.arquillian.container.managed.ManagedDeployableContainer] (main) Bundles path is deprecated and no longer used.
23:30:04,260 INFO  [org.jboss.as.arquillian.container.managed.ManagedDeployableContainer] (main) Starting container with: [C:\Java\jdk1.8.0_45\bin\java, -D[Standalone], -Dorg.jboss.ejb.client.wildfly-testsuite-hack=true, -Xmx512m, -XX:MetaspaceSize=128m, -Djboss.dist=C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering/../../../dist/target/wildfly-10.1.0.Final-SNAPSHOT, -Djava.net.preferIPv4Stack=false, -Djava.net.preferIPv6Addresses=true, -server, -Dts.timeout.factor=100, -Dnode0=::1, -Dnode1=::1, -Dmcast=ff13::1, -Dmcast.ttl=0, -Djbossas.ts.submodule.dir=C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering, -Djbossas.ts.integ.dir=C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering/.., -Djbossas.ts.dir=C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering/../.., -Djbossas.project.dir=C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering/../../.., -Djboss.inst=C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering/target/wildfly-SYNC-tcp-0, -Djboss.node.name=node-0, -ea, -Djboss.home.dir=C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0, -Dorg.jboss.boot.log.file=C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\standalone\log\server.log, -Dlogging.configuration=file:C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\standalone\configuration\logging.properties, -jar, C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\jboss-modules.jar, -mp, C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\modules;C:\BuildAgent\work\a31d203e70e89f90\dist\target\wildfly-10.1.0.Final-SNAPSHOT\modules;C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\modules, org.jboss.as.standalone, -Djboss.home.dir=C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0, -Djboss.server.base.dir=C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\standalone, -Djboss.server.log.dir=C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\standalone\log, -Djboss.server.config.dir=C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\standalone\configuration, -Dts.wildfly.version=10.1.0.Final-SNAPSHOT, -c=standalone-full-ha.xml]
23:30:04,748 INFO  [org.jboss.modules] (main) JBoss Modules version 1.5.1.Final
23:30:04,979 INFO  [org.jboss.msc] (main) JBoss MSC version 1.2.6.Final
23:30:05,063 INFO  [org.jboss.as] (MSC service thread 1-6) WFLYSRV0049: WildFly Full 10.1.0.Final-SNAPSHOT (WildFly Core 2.1.0.CR1) starting
23:30:06,630 INFO  [org.jboss.as.server] (Controller Boot Thread) WFLYSRV0039: Creating http management service using socket-binding (management-http)
23:30:06,648 INFO  [org.xnio] (MSC service thread 1-1) XNIO version 3.3.4.Final
23:30:06,656 INFO  [org.xnio.nio] (MSC service thread 1-1) XNIO NIO Implementation Version 3.3.4.Final
23:30:06,741 INFO  [org.jboss.remoting] (MSC service thread 1-1) JBoss Remoting version 4.0.18.Final
23:30:06,799 INFO  [org.jboss.as.connector.subsystems.datasources] (ServerService Thread Pool -- 39) WFLYJCA0004: Deploying JDBC-compliant driver class org.h2.Driver (version 1.3)
23:30:06,833 INFO  [org.wildfly.extension.io] (ServerService Thread Pool -- 43) WFLYIO001: Worker 'default' has auto-configured to 8 core threads with 64 task threads based on your 4 available processors
23:30:06,853 INFO  [org.jboss.as.connector] (MSC service thread 1-7) WFLYJCA0009: Starting JCA Subsystem (WildFly/IronJacamar 1.3.2.Final)
23:30:06,863 INFO  [org.wildfly.iiop.openjdk] (ServerService Thread Pool -- 45) WFLYIIOP0001: Activating IIOP Subsystem
23:30:06,880 INFO  [org.jboss.as.clustering.jgroups] (ServerService Thread Pool -- 49) WFLYCLJG0001: Activating JGroups subsystem.
23:30:06,881 INFO  [org.jboss.as.naming] (ServerService Thread Pool -- 57) WFLYNAM0001: Activating Naming Subsystem
23:30:06,899 INFO  [org.jboss.as.connector.deployers.jdbc] (MSC service thread 1-2) WFLYJCA0018: Started Driver service with driver-name = h2
23:30:06,900 WARN  [org.jboss.as.txn] (ServerService Thread Pool -- 66) WFLYTX0013: Node identifier property is set to the default value. Please make sure it is unique.
23:30:06,915 INFO  [org.jboss.as.jsf] (ServerService Thread Pool -- 52) WFLYJSF0007: Activated the following JSF Implementations: [main]
23:30:06,918 INFO  [org.jboss.as.webservices] (ServerService Thread Pool -- 68) WFLYWS0002: Activating WebServices Extension
23:30:06,949 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 44) WFLYCLINF0001: Activating Infinispan subsystem.
23:30:06,950 INFO  [org.jboss.as.security] (ServerService Thread Pool -- 64) WFLYSEC0002: Activating Security Subsystem
23:30:06,954 INFO  [org.jboss.as.naming] (MSC service thread 1-5) WFLYNAM0003: Starting Naming Service
23:30:06,961 INFO  [org.wildfly.extension.undertow] (MSC service thread 1-4) WFLYUT0003: Undertow 1.3.16.Final starting
23:30:06,974 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 67) WFLYUT0003: Undertow 1.3.16.Final starting
23:30:06,983 INFO  [org.jboss.as.mail.extension] (MSC service thread 1-3) WFLYMAIL0001: Bound mail session [java:jboss/mail/Default]
23:30:07,033 INFO  [org.jboss.as.security] (MSC service thread 1-5) WFLYSEC0001: Current PicketBox version=4.9.4.Final
23:30:07,149 INFO  [org.jboss.as.ejb3] (MSC service thread 1-4) WFLYEJB0482: Strict pool mdb-strict-max-pool is using a max instance size of 16 (per class), which is derived from the number of CPUs on this host.
23:30:07,151 INFO  [org.jboss.as.ejb3] (MSC service thread 1-6) WFLYEJB0481: Strict pool slsb-strict-max-pool is using a max instance size of 64 (per class), which is derived from thread worker pool sizing.
23:30:07,342 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 67) WFLYUT0014: Creating file handler for path 'C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0/welcome-content' with options [directory-listing: 'false', follow-symlink: 'false', case-sensitive: 'true', safe-symlink-paths: '[]']
23:30:07,399 INFO  [org.wildfly.extension.undertow] (MSC service thread 1-5) WFLYUT0012: Started server default-server.
23:30:07,410 INFO  [org.wildfly.extension.undertow] (MSC service thread 1-2) WFLYUT0018: Host default-host starting
23:30:07,553 INFO  [org.wildfly.extension.undertow] (MSC service thread 1-5) WFLYUT0006: Undertow AJP listener ajp listening on [::1]:8009
23:30:07,556 INFO  [org.wildfly.extension.undertow] (MSC service thread 1-2) WFLYUT0006: Undertow HTTP listener default listening on [::1]:8080
23:30:07,569 INFO  [org.jboss.modcluster] (ServerService Thread Pool -- 70) MODCLUSTER000001: Initializing mod_cluster version 1.3.1.Final
23:30:07,609 INFO  [org.wildfly.iiop.openjdk] (MSC service thread 1-7) WFLYIIOP0009: CORBA ORB Service started
23:30:07,617 INFO  [org.jboss.as.connector.subsystems.datasources] (MSC service thread 1-3) WFLYJCA0001: Bound data source [java:jboss/datasources/ExampleDS]
23:30:07,634 INFO  [org.jboss.modcluster] (ServerService Thread Pool -- 70) MODCLUSTER000032: Listening to proxy advertisements on /ff13:0:0:0:0:0:0:1:23364
23:30:07,795 INFO  [org.jboss.as.server.deployment.scanner] (MSC service thread 1-7) WFLYDS0013: Started FileSystemDeploymentService for directory C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\standalone\deployments
23:30:07,953 INFO  [org.jboss.ws.common.management] (MSC service thread 1-3) JBWS022052: Starting JBossWS 5.1.3.Final (Apache CXF 3.1.4) 
23:30:11,000 INFO  [org.wildfly.extension.messaging-activemq] (MSC service thread 1-8) WFLYMSGAMQ0001: AIO wasn't located on this platform, it will fall back to using pure Java NIO.
23:30:11,074 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 71) AMQ221000: live Message Broker is starting with configuration Broker Configuration (clustered=true,journalDirectory=C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\standalone\data\activemq\journal,bindingsDirectory=C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\standalone\data\activemq\bindings,largeMessagesDirectory=C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\standalone\data\activemq\largemessages,pagingDirectory=C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\standalone\data\activemq\paging)
23:30:11,111 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 71) AMQ221013: Using NIO Journal
23:30:11,295 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 71) AMQ221043: Protocol module found: [artemis-server]. Adding protocol support for: CORE
23:30:11,301 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 71) AMQ221043: Protocol module found: [artemis-amqp-protocol]. Adding protocol support for: AMQP
23:30:11,311 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 71) AMQ221043: Protocol module found: [artemis-hornetq-protocol]. Adding protocol support for: HORNETQ
23:30:11,313 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 71) AMQ221043: Protocol module found: [artemis-stomp-protocol]. Adding protocol support for: STOMP


```

--- a/ejb3/src/main/java/org/jboss/as/ejb3/component/interceptors/LoggingInterceptor.java
+++ b/ejb3/src/main/java/org/jboss/as/ejb3/component/interceptors/LoggingInterceptor.java
@@ -26,6 +26,7 @@ import java.lang.reflect.Method;
 
 import org.jboss.as.ee.component.Component;
 import org.jboss.as.ejb3.logging.EjbLogger;
+import org.jboss.as.ejb3.component.EJBComponentUnavailableException;
 import org.jboss.as.ejb3.component.EJBComponent;
 import org.jboss.as.ejb3.tx.ApplicationExceptionDetails;
 import org.jboss.invocation.ImmediateInterceptorFactory;
@@ -64,6 +65,10 @@ public class LoggingInterceptor implements Interceptor {
         try {
             // we just pass on the control and do our work only when an exception occurs
             return interceptorContext.proceed();
+        } catch ( EJBComponentUnavailableException ex) {
+            if ( EjbLogger.EJB3_INVOCATION_LOGGER.isTraceEnabled() )
+                EjbLogger.EJB3_INVOCATION_LOGGER.trace(ex.getMessage());
+            throw ex;
         } catch (Throwable t) {
             final Method invokedMethod = interceptorContext.getMethod();
             // check if it's an application exception. If yes, then *don't* log

